syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -1023,6 +1028,77 @@ def detail(worker: AbstractWorker, tensor_tuple: tuple) -> "AdditiveSharingTenso
 
         return tensor
 
+    @staticmethod
+    def bufferize(
+        worker: AbstractWorker, tensor: "AdditiveSharingTensor"
+    ) -> "AdditiveSharingTensorPB":
+        """
+        This function takes the attributes of a AdditiveSharingTensor and saves them in a protobuf object
+        Args:
+            tensor (AdditiveSharingTensor): a AdditiveSharingTensor
+        Returns:
+            protobuf: a protobuf object holding the unique attributes of the additive shared tensor
+        Examples:
+            data = protobuf(tensor)
+        """
+
+        location_ids = []
+        shares = []
+        if hasattr(tensor, "child"):
+            for key, value in tensor.child.items():
+                location_ids.append(sy.serde.protobuf.serde.create_protobuf_id(key))
+                shares.append(value)
+
+        # Don't delete the remote values of the shares at simplification
+        tensor.set_garbage_collect_data(False)
+
+        protobuf_tensor = AdditiveSharingTensorPB()
+        protobuf_tensor.id.CopyFrom(sy.serde.protobuf.serde.create_protobuf_id(tensor.id))
```

Comment:
Not sure if this will work, but it's worth a try. You can directly set child message fields on parent message fields, but I think there's some magic under the hood that creates the child messages as they're filled in. Not sure you can pass the child messages around as parameters though. I'll test it and find out.